src/Stratis.Bitcoin.Features.Consensus/ConsensusSettings.cs

```diff
@@ -58,20 +58,27 @@ private ConsensusSettings LoadFromConfig()
         }
 
         /// <summary>
-        /// Gets the default setting for the hash for the block to assume is valid.
+        /// Gets the default setting for the block hash of the block to assume is valid.
         /// </summary>
         /// <param name="network">Network to return hash for.</param>
         /// <returns>The hash for the assume valid block.</returns>
         private uint256 GetDefaultAssumeValidBlock(Network network)
         {
-            // TODO: Figure out where to put defaults so they are easy to configure.
             uint256 defaultAssumeValidBlock = null;
-            if (!network.IsBitcoin() && network.IsTest())
+            if (network.IsBitcoin())
```

Comment:
This is ok for this PR but how about we put it in Network? (its now part of the project so should be easy)

Reply:
Sure. I also realized that bitcoin core puts in what would be our ConsensusOptions. Do we have a different set of ConsensusOptions for each network?

Reply:
Sort of yes we can set stuff in our ConsensusOptions when building the network

Reply:
Do you think better in network or consensusOptions?